app/src/main/java/com/example/mycalculator/mvp/model/CalculatorModel.java

```diff
@@ -4,14 +4,65 @@
 
 public class CalculatorModel implements CalculatorContract.CalculatorModelContract {
 
-    private String firstOperand;
+    private String firstOperand = "";
```

Comment:
here you can use the EMPTY_STRING constant

Reply:
Done. Thx